chacha20poly1305/Cargo.toml

```diff
@@ -23,7 +23,7 @@ aead = { version = "=0.6.0-pre.0", default-features = false }
 chacha20 = { version = "=0.10.0-pre", features = ["zeroize"] }
 cipher = "=0.5.0-pre.4"
 poly1305 = "=0.9.0-pre"
-zeroize = { version = "1.6", default-features = false }
+zeroize = { version = "1.8", default-features = false }
```

Comment:
I don't think we need this bump. It's better to just update Cargo.lock.

Reply:
Good catch